aws-common-runtime/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set(AWS_DEPS_DOWNLOAD_DIR "${AWS_DEPS_BUILD_DIR}/downloads" CACHE PATH "Dependen
 
 message("install dir ${AWS_DEPS_INSTALL_DIR}")
 set(AWS_C_COMMON_URL "https://github.com/awslabs/aws-c-common.git")
-set(AWS_C_COMMON_SHA "v0.3.11")
+set(AWS_C_COMMON_SHA "v0.3.13")
```

Comment:
There's now a v0.3.14, which includes fixes for callstack decoding tests on linux. Should be a trivial update.